test/cql-pytest/test_permissions.py

```diff
@@ -164,27 +164,33 @@ def test_udf_permissions_serialization(cql):
 # are properly handled, with right permissions granted.
 # Cassandra doesn't quote names properly, so the test fails
 def test_udf_permissions_quoted_names(cassandra_bug, cql):
-    schema = "a int primary key"
+    udt_name = '"weird_udt[t^t]a^b^[]"'
```

Comment:
It's better to always use unique names, in case we ever have a problem with killed test, parallel tests, etc. For example you can do something like '{unique_name()}"weird_udt[t^t]a^b^[]"'.